airflow/contrib/hooks/jenkins_hook.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from airflow.hooks.base_hook import BaseHook
+
+import jenkins
+
+
+class JenkinsHook(BaseHook):
+    '''
+    Hook to manage connection to jenkins server
+    '''
+
+    def __init__(self, conn_id='jenkins_default'):
+        connection = self.get_connection(conn_id)
+        print(connection)
+        self.connection = connection
+        print ('Trying to connect to http://%s:%d' % (connection.host, connection.port))
```

Comment:
Remove print.

Reply:
Should I transform this last one in a logging.info/debug ? It can be useful to have the full url used in case of error when configuring the connection.

Reply:
In this case yes please convert to logging